pyproject.toml

```diff
@@ -6,19 +6,3 @@ requires = [
 ]
 build-backend = 'setuptools.build_meta'
 
-dependencies = [
-    'numpy >= 1.20',
-    'prettytable>=3.9.0',
-    'pydantic>=2.7.2',
-    'StrEnum>=0.4.15; python_version<"3.11"',
-]
-
-[optional-dependencies]
-Dev = ['pytest>=7.4.0', 'pytest-cov>=4.1.0']
-Matlab_latest = ['matlabengine']
-Matlab-2023b = ['matlabengine==23.2.1']
-Matlab-2023a = ['matlabengine==9.14.3']
-Matlab-2022b = ['matlabengine==9.13.9']
-Matlab-2022a = ['matlabengine==9.12.19']
-Matlab-2021b = ['matlabengine==9.11.21']
-Matlab-2021a = ['matlabengine==9.10.3']
```

Comment:
Are these dependencies accounted for anywhere else seeing as they're removed from here?

Reply:
Yes, its all in setup.py. The build was failing because the pyproject.toml was not fully defined so I opted to scrap